pkgs/by-name/c-/c-periphery/package.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "c-periphery";
+  version = "2.4.2";
+
+  src = fetchFromGitHub {
+    owner = "vsergeev";
+    repo = "c-periphery";
+    rev = "v${version}";
```

nit: consider using finalAttrs

```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "c-periphery";
  version = "2.4.2";

  src = fetchFromGitHub {
    owner = "vsergeev";
    repo = "c-periphery";
    rev = "v${finalAttrs.version}";
```